mise-en-page/accueil-institution-directives.html

```diff
@@ -167,7 +164,8 @@ <h3>Image</h3>
 									<p>La zone de 1200x726 derrière le H1 peut être utilisée pour une bannière ou une image. L'image n'a pas besoin d'occuper tout l'espace et peut être personnalisée au besoin.</p>
 									<p>L'image et le H1 doivent être visuellement distincts de la <a href="https://www.canada.ca/">page d'accueil de Canada.ca</a> afin d'éviter toute confusion entre la page d'accueil de l'institution et celle de Canada.ca.</p>
 									<p>Si vous n'utilisez pas d'image, l'arrière-plan devrait être blanc.</p>
-									<p><b>Variation&nbsp;:</b> Appliquez la classe <code>ilp-cover-hide-xxs</code> ou la classe <code>ilp-cover-hide-xs</code> en plus de <code>ilp-cover-img</code> pour cacher l'image pour les appareils mobiles.</p>
+									<p><b>Variation&nbsp;:</b> The class <code>bg-hide-sm</code> hides the image on tablet, <code>bg-hide-xs</code> hides the image on mobile and
+									<code>bg-right</code> positions the image on the right.</p>
```

Comment:
Will this be translated in French later? Do you need some help for that?

Reply:
Yes it's being previewed by Emilie and won't be merged until it's all ready